### PR TITLE
New 7.3 test for Audible Cues

### LINKED ISSUE #378: Create new test 7.3 Test for audible cues
1.1.1 Non-Text requires a text alternative that serves the equivalent purpose for meaningful sounds (beeps, chimes, etc.).

Baseline 16: Update Exceptions Audio-only to remove "information" from last bullet.

### PR DESCRIPTION
**Pull request by kengdoj**
Closes #378

### CODE REVIEW

File: _baselines/07Sensory.md [hunk 1]
```diff
@@ -62,6 +64,25 @@ Users affected by this requirement are not limited to users of assistive technol
 #### Test Results
 <p id="2TR">If any of the above checks fail, then Baseline Test 7.2-SensoryCharacteristics fails.</p>
 
+### 7.3 Test Procedure for Audible Cues
+**Baseline Test ID:** 7.3-AudibleCues
+#### Identify Content
+<p id="3IC">Identify any short sound/audible cue that serves as a notification to the user, such as a beep that signifies an error has occurred.</p>
```
Comment (cliffbert) [inline _baselines/07Sensory.md:70]: what's the word for longer than a beep ... chime? ring?

Comment (kengdoj) [reply _baselines/07Sensory.md:70]:
```suggestion
<p id="3IC">Identify any short sound/audible cue that serves as a notification to the user, such as a beep that signifies an error has occurred or a chime to indicate a new incoming message.</p>
```

File: _baselines/07Sensory.md [hunk 2]
```diff
@@ -62,6 +64,25 @@ Users affected by this requirement are not limited to users of assistive technol
...
 <p id="2TR">If any of the above checks fail, then Baseline Test 7.2-SensoryCharacteristics fails.</p>
 
+### 7.3 Test Procedure for Audible Cues
+**Baseline Test ID:** 7.3-AudibleCues
+#### Identify Content
+<p id="3IC">Identify any short sound/audible cue that serves as a notification to the user, such as a beep that signifies an error has occurred or a chime to indicate a new incoming message.</p>
+
+#### Test Instructions
+<ol id="3TI">
+    <li id="3TI-1">Check that a text alternative that describe's the purpose of the sound is provided with the audible cue. [SC 1.1.1]<br>
```
Comment (kengdoj) [inline _baselines/07Sensory.md:74]:
```suggestion
    <li id="3TI-1">Check that a text alternative that describes the purpose of the sound is provided with the audible cue. [SC 1.1.1]<br>
```

File: _baselines/07Sensory.md [hunk 3]
```diff
@@ -62,6 +64,25 @@ Users affected by this requirement are not limited to users of assistive technol
 #### Test Results
 <p id="2TR">If any of the above checks fail, then Baseline Test 7.2-SensoryCharacteristics fails.</p>
 
+### 7.3 Test Procedure for Audible Cues
+**Baseline Test ID:** 7.3-AudibleCues
+#### Identify Content
+<p id="3IC">Identify any short sound/audible cue that serves as a notification to the user, such as a beep that signifies an error has occurred or a chime to indicate a new incoming message.</p>
```
Comment (kengdoj) [inline _baselines/07Sensory.md:70]:
```suggestion
<p id="3IC">Identify any short sound/audible cue that serves as a notification to the user, such as a beep that signifies an error has occurred or a chime to indicate an incoming message.</p>
```

**Review by cliffbert — APPROVED** [review]

**Pull request merged**